app/Http/Controllers/Api/ApiController.php

```diff
@@ -2,126 +2,10 @@
 
 namespace Rogue\Http\Controllers\Api;
 
-use League\Fractal\Manager;
-use League\Fractal\Resource\Item;
-use League\Fractal\Resource\Collection;
-use League\Fractal\Serializer\DataArraySerializer;
 use Rogue\Http\Controllers\Traits\FiltersRequests;
 use Rogue\Http\Controllers\Controller as BaseController;
-use League\Fractal\Pagination\IlluminatePaginatorAdapter;
 
 class ApiController extends BaseController
 {
     use FiltersRequests;
-
-    /**
-     * @var League\Fractal\Manager
-     */
-    protected $manager;
```

Comment:
Is removing all of this going to break any of our other api routes that use this controller?